pkg/cmd/server/start/master_args.go

```diff
@@ -44,6 +44,8 @@ type MasterArgs struct {
 	// or URL). Defaults to same as --master.
 	MasterPublicAddr flagtypes.Addr
 
+	StartAPI         bool
+	StartControllers bool
```

Comment:
Suggestion: Common usage of `NewDefaultMasterArgs` followed by setting the `Start*` bits individually to `false` requires knowledge of all the flags.  Having `NewDefaultMasterArgs` return with the `Start*` bits disabled, will keep things like `openshift start master api` operating as intended if we add more switches.